### DISCUSSION TITLE
Can /premium require Rejoin Instead of redo Command?

### DISCUSSION THREAD

**Discussion by BlueTree242** [games647/FastLogin#646] [Q&A] [answered]
So, if player does `/premium` 2 times then later joins and he is not even a premium player... he can't. some players may use commands without knowing what they are doing (even confirming). So can the player get kicked and when they rejoin with authentication if they success they are added to premium, else the proccess failes?

**Comment by TuxCoding** [comment, marked as answer]
There is already a feature request for that.

**Reply by BlueTree242** [reply]
> There is already a feature request for that.

what is the config option name?

**Reply by TuxCoding** [reply]
None there is a request only for that.

**Reply by BlueTree242** [reply]
> None there is a request only for that.

i think its time  to prepare to PR then

**Comment by applegamerho** [comment]
did you solve it?
i have the same question...
